pkg/operator/ceph/object/objectstore.go

```diff
@@ -289,6 +290,9 @@ func createSimilarPools(context *Context, pools []string, poolSpec model.Pool) e
 			if err != nil {
 				return errors.Errorf("failed to create pool %s for object store %s", name, context.Name)
 			}
+		} else {
+			//update the pools if it already exist
```

Comment:
```suggestion
			// update the pool if it already exists
```

Reply:
Thanks, Please check the commit message and let me know if it's correct and explains the commit clearly.